### PR TITLE
Created Rowan-Loft's Final Project Idea and Plan

### PR DESCRIPTION
**Pull request by Rowan-Loft**
This is my initial idea and plan for the final project which looks at creating a user-input menu for navigating messy csv data pertaining to commercial airports in the United States.

### CODE REVIEW

File: _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+author: Rowan-Loft
+title: Rowan-Loft's Initial Idea & Work Plan
+---
+
+For my final project, I will be looking to create a user-navigable program for a sample size of public commercial and (tentatively) non-commercial aiports in the United States. I say a sample size because there are over 5,000 public airports, and this program will specifically not include all other airports in the US, as the number jumps to just under 25,000.
```
Comment (hannahmoutran) [inline _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md:7]: line 7 typo: airports -Hannah

File: _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+There is already a present and up to date .csv containing data that provides information on open airports, closed airports, and helipads in the US that is produced by the United Nations Office for the Coordination of Human Affairs (OCHA), The list will have to be trunkated to some extent as I'm not interested in closed airports and helipads, which contribute to the files' just-under 30,000 rows of data. Further cutting will have to be done to account for some information that would not be pertinent to someone navigating the program such as elevation, longitude, and latitude of an airport.
```
Comment (hannahmoutran) [inline _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md:9]: typo: '.' after (OCHA)-Hannah

Comment (hannahmoutran) [reply _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md:9]: Also: truncated

**Comment by hannahmoutran** [comment]
Hi Tyler!  I looked over your post, looks great except for a few typos.  Let me know when you're ready for me to approve!        -Hannah

File: _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md (hunk 1 above)
Comment (Rowan-Loft) [reply _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md]: Fixed.

File: _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md (hunk 2 above)
Comment (Rowan-Loft) [reply _posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md]: Fixed and fixed. Thanks for keeping an eye out.

**Review by Prakash-sa — APPROVED** [review]
Great work Rowan!

**Pull request merged**